src/style.css

```diff
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
You have not provided a fallback font-family. If the main one doesn't work, the browser will use a default one which may not fit your design.